routes/api.php

```diff
@@ -16,3 +16,14 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+// Third Party Integration
+$router->group(['prefix' => 'api/v1', 'middleware' => ['third-party-api-token-auth']], function() {
```

Comment:
The `routes/api.php` file is [already prefixed](https://github.com/DoSomething/chompy/blob/85b19765da65305192fea67f199642c1b980b4b2/app/Providers/RouteServiceProvider.php#L66-L72) with `api/`, so this should just read `'prefix' => 'v1'`.

Reply:
omg THANK YOU!!